skbio/sequence/_sequence.py

```diff
@@ -1230,21 +1303,55 @@ def copy(self, deep=False):
         2   []        8
         3   []        5
 
+        >>> interval.metadata['gene'] = ['BCC7']
+        >>> seq.interval_metadata   # doctest: +ELLIPSIS
+        1 interval feature
+        ------------------
+        Interval(interval_metadata=<...>, bounds=[(0, 2)], \
+fuzzy=[(False, False)], metadata={'gene': ['BCC7']})
+        >>> interval.bounds = [(1, 3)]
+        >>> seq.interval_metadata  # doctest: +ELLIPSIS
+        1 interval feature
+        ------------------
+        Interval(interval_metadata=<...>, bounds=[(1, 3)], \
+fuzzy=[(False, False)], metadata={'gene': ['BCC7']})
+        >>> seq_copy.interval_metadata   # doctest: +ELLIPSIS
+        1 interval feature
+        ------------------
+        Interval(interval_metadata=<...>, bounds=[(0, 2)], \
+fuzzy=[(False, False)], metadata={'gene': 'p53'})
+
+        >>> interval2 = seq_copy.interval_metadata.add(
+        ...     [(2, 3)], metadata={'gene': 'brca1'})
+        >>> seq_copy.interval_metadata   # doctest: +ELLIPSIS
+        2 interval features
+        -------------------
+        Interval(interval_metadata=<...>, bounds=[(0, 2)], \
+fuzzy=[(False, False)], metadata={'gene': 'p53'})
+        Interval(interval_metadata=<...>, bounds=[(2, 3)], \
+fuzzy=[(False, False)], metadata={'gene': 'brca1'})
+        >>> seq.interval_metadata  # doctest: +ELLIPSIS
+        1 interval feature
+        ------------------
+        Interval(interval_metadata=<...>, bounds=[(1, 3)], \
+fuzzy=[(False, False)], metadata={'gene': ['BCC7']})
+
         Since only a *shallow* copy was made, updates to mutable objects stored
         as metadata affect the original sequence's metadata:
 
+        >>> seq_copy = seq.copy()
```

Comment:
What was the reason for adding this line? `seq_copy` did already exist and it seems like the prose would still make sense without this line.